### DISCUSSION TITLE
[Question] Bossfight Item drops

### DISCUSSION THREAD

**Discussion by Chance4us** [diasurgical/DevilutionX#3430] [General]
Question not an issue. I have questions about "Warlord of Blood" and "Diablo" item drops.

1. I just saw that the opposing "Warlord of Blood" dropped a Book of Chain Lightning, but no magical item.
Is it normal that he can drop books?

2. I've seen Diablo drop either gold or items many times when killed. But you can no longer collect the item because then the game is over. Is it normal for him to drop something?

Thanks in advance.

**Comment by qndel** [comment]
If not an issue, open discussion instead :( 
2. Diablo is a normal monster so ofc he drops loot
1. Can't find it in the code but they drop magic items or books, dunno if it's coded like this or a book is treated like a magic item

**Reply by Chance4us** [reply]
Thanks. I will open a discussion next time if I have any questions.

**Comment by AJenbo** [comment]
You can pick up the items Diablo drops by quickly saving and reloading as that will let you walk around and pick up items. Then save again before the movie start and you will have the item in the next game.
